README.md

```diff
@@ -95,17 +95,17 @@ Skaffold supports the following tools:
   * Jib Maven/Gradle locally
 * Test
   * with container-structure-test
-* Deploy
-  * Kubernetes Command-Line Interface (`kubectl`)
-  * Helm
-  * kustomize
 * Tag
   * tag by git commit
   * tag by current date&time
   * tag by environment variables based template
 * Push
   * don't push - keep the image on the local daemon
   * push to registry
+* Deploy
+  * Kubernetes Command-Line Interface (`kubectl`)
+  * Helm
+  * kustomiz
```

Comment:
Should be `kustomize`

Reply:
Sorry for causing the issue, I will more carefully.